core/model/src/payment.rs

```diff
@@ -130,10 +130,19 @@ pub mod local {
     #[error("")]
     pub struct NoError {} // This is needed because () doesn't implement Display
 
+    #[derive(Clone, Debug, Serialize, Deserialize, Default, derive_more::Display)]
+    #[display(fmt = "{}-{}-{}", driver, network, "token.to_lowercase()")]
+    pub struct Platform {
```

Comment:
Payment platform is **not** required to have this format. If it was the case, the token->platform mapping wouldn't be necessary at all. Also, this will cause conflicts with `payment/multi-network` branch.

Please remove this struct and revert all changes that include it.